conn/ssl.c

```diff
@@ -740,7 +740,7 @@ static int tls_close(struct Connection *conn)
 static bool check_certificate_cache(X509 *peercert)
 {
   unsigned char peermd[EVP_MAX_MD_SIZE];
-  unsigned int peermdlen;
+  size_t peermdlen;
```

Comment:
`X509_digest()` is expecting `unsigned int *`
When might `size_t` and `unsigned int` differ in size?

Reply:
As far as I can tell, there seems to be no risk in going from an `unsigned int` to `size_t`. Going the other way would be riskier as `unsigned int` is more platform-dependent in its size.